connector_magento/data/connector_magento_data.xml

```diff
@@ -0,0 +1,140 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+        <record id="product.decimal_discount" model="decimal.precision">
+            <field name="digits">3</field>
+        </record>
+
+        <record model="ir.cron" id="ir_cron_import_sale_orders" forcecreate="True">
+            <field name="name">Magento - Import Sales Orders</field>
+            <field eval="False" name="active"/>
+            <field name="user_id" ref="base.user_root"/>
+            <field name="interval_number">1</field>
+            <field name="interval_type">days</field>
+            <field name="numbercall">-1</field>
+            <field eval="False" name="doall"/>
+            <field ref="connector_magento.model_magento_backend" name="model_id"/>
+            <field eval="'_scheduler_import_sale_orders'" name="function"/>
+            <field eval="'()'" name="args"/>
```

Comment:
change to 
<field name="state">code</field>
<field name="code">model._scheduler_import_sale_orders()</field>